analysis/aws_rules_vpc_flow_logs/aws_vpc_inbound_traffic_port_blacklist.py

```diff
@@ -6,6 +6,10 @@
 
 
 def rule(event):
+    # This rule can only be evaluated if users have enabled these fields in their VPC Flow Logs
```

Comment:
can we do the `.get()` pattern for consistency?

Reply:
Fair enough, it adds a bit more unintuitive logic though so I added some comments that hopefully clarify.